pkgs/applications/editors/rstudio/default.nix

```diff
@@ -30,6 +31,9 @@
 , nodejs
 , mkYarnModules
 , qmake
+, server ? false # build server version
+, sqlite
```

Comment:
what happens with these dependencies if server is false? I assume everything is lazy and so the arguments aren't evaluated, so pam could be null in the argument list and not cause an issue. Is that correct?

Reply:
sorry, not null, I mean not defined (and by argument list I meant attr set I think)

Reply:
No, it will be an error if undefined as pattern matching isn't lazy. This is only going to be a problem if something is removed from nixpkgs, at which point we need to fix this expression anyway.

Reply:
makes sense, thanks!